components/meter/test/meter-circle.visual-diff.html

```diff
@@ -26,6 +26,9 @@
 	<div class="visual-diff" style="width: 90px;">
 		<d2l-meter-circle id="round-to-zero" value="0.004" max="100"></d2l-meter-circle>
 	</div>
+	<div class="visual-diff" style="width: 90px;">
+		<d2l-meter-circle id="zero-max" value="10" max="0"></d2l-meter-circle>
+	</div>
```

Comment:
So there's already a "max-zero" test where the value is also 0. Could you rename it to "max-zero-value-zero" and name this one something like "max-zero-with-value" just to make it clear what the difference is? You should be able to just rename the existing visual diff file(s) and it'll "just work" when the diff runs again.

Reply:
That makes sense to me, I have renamed the tests as per your suggestion. When I checked the diffs looked as expected, I will double check when the visual diff test runs again